backend/videoService/tests/opentokVideoService.test.ts

```diff
@@ -0,0 +1,118 @@
+import { describe, expect, it, jest } from '@jest/globals';
```

Comment:
Note: if you notice some differences between this test file and `vonageVideoService.test.ts`, no you don't 😆 
just kidding - there are some slight difference since the `opentok` package uses callback and exports itself without a `default` but with `module.exports = OpenTok`.

also you'll notice that in this file I'm also mocking `axios` - that is because we have to use `axios` in `opentokVideoService` but not `vonageVideoService` since the `opentok` library does not support `enableCaptions` and `disableCaptions` methods so we have to do direct REST API requests. I've left some comments about it in the codebase, too.

Reply:
Well, to be fair, for tests we prefer WET over DRY ;-)